omxplayer/bus_finder.py

```diff
@@ -20,7 +21,24 @@ def get_address(self):
         return self.address
 
     def wait_for_file(self):
-        while not os.path.isfile(self.path):
-            time.sleep(0.05)
+        if self.path:
+            # Wait for the given path to exist.
```

Comment:
Avoid using comments like this, if it needs commenting, it should be in a separate method.
e.g. `wait_for_dbus_path`